src/views/Room.vue

```diff
@@ -48,6 +48,7 @@ export default {
       peers: [],
       localPeer: null,
       infoPage: null,
+      signalingConnectedBefore = false,
```

Comment:
Syntax error

```suggestion
      signalingConnectedBefore: false,
```

Reply:
I am not that familiar with javascript. But could we apply some sanitizer/compiler to our javascript files. Such that these kind of syntax errors could be detected in CI? I could setup github actions for this repository.

Reply:
@erikzenker It is already happening and it would be super duper to have such a github action. The command is `$ yarn lint`